src/profile.ts

```diff
@@ -0,0 +1,142 @@
+import * as path from 'path';
```

Comment:
Doesn't a lot of this code already exist in an internal module? What was the reason to reimplement this logic?

Reply:
It didn't exist in an internal module; this isn't how `fuse profiles` are loaded.